src/text-field.js

```diff
@@ -30,81 +29,58 @@ class Textfield extends React.Component {
   }
 
   render(){
-
     const {
-      className,
-      type,
-      label,
-      floatingLabel,
-      error,
+      children,
       expandable,
-      button,
       icon,
-      expandableHolder
-    } = this.props;
-
-    let {
-      id
+      floating,
+      className,
     } = this.props;
 
-    const outerClasses = classnames(outerBaseClasses, {
-      'mdl-textfield--floating-label': floatingLabel,
+    const inputChildren = React.Children.map(this.props.children, (child) => {
```

Comment:
use `children` directly (instead of this.props.children) since you destructured it but didn't use it